modules/dev-tools/scripts/build.sh

```diff
@@ -5,6 +5,7 @@ set -e
 DEV_TOOLS_DIR=`node -e "require('ocular-dev-tools/node/module-dir')()"`
 CONFIG=`node $DEV_TOOLS_DIR/node/get-config.js ".babel.configPath"`
 MODULES=`node $DEV_TOOLS_DIR/node/get-config.js ".modules" | sed -E "s/,/ /g"`
+EXTENSIONS=".es6,.js,.es,.jsx,.mjs"
```

Comment:
This is the current `@babel/cli` default - it was easier to specify here than leave out.